core/src/utils/overlays.ts

```diff
@@ -496,7 +496,7 @@ export const present = async <OverlayPresentOptions>(
    * from returning focus as a result.
    */
   if (overlay.el.tagName !== 'ION-TOAST') {
-    focusPreviousElementOnDismiss(overlay.el);
+    focusPreviousElementOnDismissIfNeeded(overlay.el);
```

Comment:
Maybe we can call this `restoreElementFocus`? `focusPreviousElementOnDismissIfNeeded` is a bit wordy.

Reply:
Done!